cluster/examples/workloads/kubernetes/wordpress/aws/app.yaml

```diff
@@ -6,6 +6,8 @@ metadata:
   namespace: complex
 spec:
   classRef:
+    kind: RDSInstanceClass
```

Comment:
shouldn't this binding introduced by a policy? I think keeping `app.yaml` in `wordpress` directory shows the power of Crossplane, where you could apply the same workload into multiple providers without any change in the workload.

Reply:
I agree that this is a valuable feature of Crossplane! However, the scope of this PR was not to introduce defaulting behavior. I think it would be useful to demonstrate `Policy` with this example, as well as with many of the others. That work is being tracked by #588. Feel free to add a comment on there if you believe that it does not encompass this specifically.

Reply:
Sure, it can be done later. However keep in mind that having two copies of essentially same yaml file is also prone to inconsistency over time.

In that case, please update a few `.md` files which track the location of `app.yaml` and `cluster.yaml` files

Reply:
Note: after further offline discussion we decided it would be useful to go ahead and update the script in this PR even though the docs will not match exactly.

Thanks for this feedback @soorena776 :)